packages/mjml-cli/README.md

```diff
@@ -77,28 +77,12 @@ $> mjml input.mjml --config.minifyOptions='{"minifyCSS": true, "removeEmptyAttri
 
 The defaults are "collapseWhitespace": true, "minifyCSS": false, "removeEmptyAttributes": true
 
-### Preserve specific tags when using inline mj-style
```

Comment:
Well this block should stay

Reply:
Sorry for taking so long to respond. I aspire to answer faster.

I have no reason to disagree with any of these. At the same time, I recommend careful review of my output.
1) I don't recall having made these changes.
- Maybe the second blank line at the end of mjml-raw/README.md. I didn't know that was important. I had some changes not directly related to `mjml-` to `mj-` in a draft of my PR, but intended to delete all those. I should have deleted this one.
2) I've compared the current `next` to `9d9e2454` (the commit immediately before my release). Where (like the above `### Preserve specific tags when using inline mj-style`) is absent from 'next' and in '9d', I'm doing copy-and-paste to put the '9d' version in my commit. If there has been a later change to any of the copy-and-paste text, I won't catch it. Maybe there's a better way to recover from this condition. (Don't know about that.)

Action: Add the `9d` version of the `### Preserve specific tags when using inline mj-style` block to `next`.

Note: Either I missed a change in `mjml-button/README.md` or was lost.

Action: Changed `mjml-button` to `mj-button` on line 1, consistent with the intent of the original PR.